### DISCUSSION TITLE
'search_phase_execution_exception', 'No mapping found for [@timestamp] in order to sort on'

### DISCUSSION THREAD

**Discussion by k4lii** [jertel/elastalert2#151] [Q&A] [answered]
<img width="756" alt="Capture" src="https://user-images.githubusercontent.com/71388385/117791449-a1fd7b00-b24a-11eb-9d3f-29aecae61447.PNG">
got an error since today, all was working fine before, the timestamp in ES7 is shift of 10h but its working

**Comment by jertel** [comment, marked as answer]
Seems that either your index for today is empty, or something else has changed with the creation or ingest of data into the new daily index.

From the Helm chart [README](https://github.com/jertel/elastalert2/tree/master/chart/elastalert2):

![image](https://user-images.githubusercontent.com/16639611/117812008-3d80f280-b22f-11eb-93b5-bb1fa67a795a.png)

**Reply by jertel** [reply]
Moved to discussion since original post did not satisfy the requirements needed to open an issue.

**Reply by k4lii** [reply]
OH ty ;) great work btw !
